.github/workflows/download_data.yml

```diff
@@ -41,25 +46,33 @@ on:
         value: ${{ jobs.download.outputs.cache_key }}
   workflow_dispatch:
     inputs:
+      url:
+        description: URL to gzip file
+        type: string
+        required: false
+        default: https://stsci.box.com/shared/static/0dt9z6b927iqgtify2a4cvls9hvapi6k.gz
       minimal:
-        description: minimal dataset
+        description: dataset is minimal (as opposed to full)
         type: boolean
         required: false
         default: true
   schedule:
     - cron: "0 0 * * 0"
   release:
+  push:
+    branches:
+      - develop
 
 env:
-  DATA_URL: https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz
+  FULL_DATA_URL: https://stsci.box.com/shared/static/qxpiaxsjwo15ml6m4pkhtk36c9jgj70k.gz
```

Comment:
FULL_DATA_URL not needed, but I guess it makes sense to keep it in case we switch anything down the line